analysis/performance_tests.py

```diff
@@ -198,26 +198,28 @@ def save(self, summary, filename):
 
 if __name__ == "__main__":
 
-#    tests = [test_small_fraction_changes_sparse,
-#             test_large_fraction_constant_sparse]
-
-#    for test in tests:
-#        testcase = test(num_transactions=[50, 100, 500, 1000, 5000, 10000],
-#                        exponents=[14],
-#                        compression=[None,])
-#        summary, msg = testcase.create_files(versions=True)
-#        testcase.save(summary, f"{testcase.testname}")
-#        summary, msg = testcase.create_files(versions=False)
-#        testcase.save(summary, f"{testcase.testname}_no_versions")
-
-    tests = [test_mostly_appends_dense]
+    # tests = [test_small_fraction_changes_sparse,
+    #          test_large_fraction_constant_sparse]
+    # for test in tests:
+    #    testcase = test(num_transactions=[50, 100, 500, 1000, 5000, 10000],
+    #                    exponents=[14],
+    #                    compression=[None,])
+    #    summary, msg = testcase.create_files(versions=True)
+    #    testcase.save(summary, f"{testcase.testname}")
+    #    summary, msg = testcase.create_files(versions=False)
+    #    testcase.save(summary, f"{testcase.testname}_no_versions")
```

Comment:
Can this code justy be removed?

Reply:
I'll do it in another PR, sorry about that.